lms/djangoapps/grades/models.py

```diff
@@ -172,13 +172,10 @@ def create(self, **kwargs):
             kwargs['course_id'] = kwargs['usage_key'].course_key
 
         visible_blocks_hash = VisibleBlocks.objects.hash_from_blockrecords(blocks=visible_blocks)
-        grade = self.model(
+        return super(PersistentSubsectionGradeQuerySet, self).create(
```

Comment:
This set of changes saves us a few sql queries by skipping the "does this exist before I try to insert it?" check, which seem reasonable in a `create` method.

Before:
```
QUERY = u'SAVEPOINT "s140067648034560_x479"' - PARAMS = ()

QUERY = u'SELECT "grades_visibleblocks"."id", "grades_visibleblocks"."blocks_json", "grades_visibleblocks"."hashed" FROM "grades_visibleblocks" WHERE "grades_visibleblocks"."hashed" = %s' - PARAMS = (u'rem4VMp2KovYEbU/Y0Tv1hJZQog=',)

QUERY = u'SAVEPOINT "s140067648034560_x480"' - PARAMS = ()

QUERY = u'INSERT INTO "grades_visibleblocks" ("blocks_json", "hashed") VALUES (%s, %s)' - PARAMS = (u'[{"locator":"i4x://org.10/course_10/problem/Test_Problem","max_score":1,"weight":null}]', u'rem4VMp2KovYEbU/Y0Tv1hJZQog=')

QUERY = u'RELEASE SAVEPOINT "s140067648034560_x480"' - PARAMS = ()

QUERY = u'SELECT (1) AS "a" FROM "grades_visibleblocks" WHERE "grades_visibleblocks"."hashed" = %s LIMIT 1' - PARAMS = (u'rem4VMp2KovYEbU/Y0Tv1hJZQog=',)

QUERY = u'SELECT (1) AS "a" FROM "grades_persistentsubsectiongrade" WHERE ("grades_persistentsubsectiongrade"."course_id" = %s AND "grades_persistentsubsectiongrade"."usage_key" = %s AND "grades_persistentsubsectiongrade"."user_id" = %s) LIMIT 1' - PARAMS = (u'org.10/course_10/Run_10', u'i4x://org.10/course_10/sequential/Test_Sequential_1', 1)

QUERY = u'INSERT INTO "grades_persistentsubsectiongrade" ("created", "modified", "user_id", "course_id", "usage_key", "subtree_edited_timestamp", "course_version", "earned_all", "possible_all", "earned_graded", "possible_graded", "visible_blocks_hash") VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)' - PARAMS = (u'2016-08-23 18:15:01.355629', u'2016-08-23 18:15:01.357589', 1, u'org.10/course_10/Run_10', u'i4x://org.10/course_10/sequential/Test_Sequential_1', u'2016-08-23 18:15:00.823819', u'', 0.0, 1.0, 0.0, 1.0, u'rem4VMp2KovYEbU/Y0Tv1hJZQog=')

QUERY = u'RELEASE SAVEPOINT "s140067648034560_x479"' - PARAMS = ()
```

After:
```
QUERY = u'SAVEPOINT "s139878613460736_x450"' - PARAMS = ()

QUERY = u'SELECT "grades_visibleblocks"."id", "grades_visibleblocks"."blocks_json", "grades_visibleblocks"."hashed" FROM "grades_visibleblocks" WHERE "grades_visibleblocks"."hashed" = %s' - PARAMS = (u'rem4VMp2KovYEbU/Y0Tv1hJZQog=',)

QUERY = u'SAVEPOINT "s139878613460736_x451"' - PARAMS = ()

QUERY = u'INSERT INTO "grades_visibleblocks" ("blocks_json", "hashed") VALUES (%s, %s)' - PARAMS = (u'[{"locator":"i4x://org.10/course_10/problem/Test_Problem","max_score":1,"weight":null}]', u'rem4VMp2KovYEbU/Y0Tv1hJZQog=')

QUERY = u'RELEASE SAVEPOINT "s139878613460736_x451"' - PARAMS = ()

QUERY = u'INSERT INTO "grades_persistentsubsectiongrade" ("created", "modified", "user_id", "course_id", "usage_key", "subtree_edited_timestamp", "course_version", "earned_all", "possible_all", "earned_graded", "possible_graded", "visible_blocks_hash") VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)' - PARAMS = (u'2016-08-23 19:43:25.498337', u'2016-08-23 19:43:25.498967', 1, u'org.10/course_10/Run_10', u'i4x://org.10/course_10/sequential/Test_Sequential_1', u'2016-08-23 19:43:25.190722', u'', 0.0, 1.0, 0.0, 1.0, u'rem4VMp2KovYEbU/Y0Tv1hJZQog=')

QUERY = u'RELEASE SAVEPOINT "s139878613460736_x450"' - PARAMS = ()
```